docs/integrations/angular.rst

```diff
@@ -110,3 +110,26 @@ Inside your main Angular application module, you need to declare `ngRaven` as a
       'myAppControllers',
       'myAppFilters'
     ]);
+
+Module loaders (CommonJS)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The raven angular module can be loaded using a module loader like Browserify or Webpack.
+
+.. code-block:: javascript
+    var angular = require('angular');
+    var ngRaven = require('raven-js/plugins/angular').module;
+    var ngRoute = require('angular-route');
+
+    var myAppFilters = require('./myAppFilters');
+    var myAppControllers = require('./myAppControllers');
+    var module = 'myApp'
```

Comment:
ah it won't be changed in the 'Show outdated' view, you need to see the whole change set again.